libs/api-interfaces/src/lib/aggregate-views/view-models/user-management/coscrad-contributor/coscrad-contributor.view-model.interface.ts

```diff
@@ -0,0 +1,13 @@
+import { IBaseViewModel } from '../../base.view-model.interface';
+
+export interface ICoscradContributorViewModel extends IBaseViewModel {
+    label: string;
+
+    description: string;
+
+    fullName: string;
+
+    // dateOfBirth?: CoscradDate;
+
+    shortBio?: string;
+}
```

Comment:
You'll need a constructor that takes in a `Contributor` and builds the corresponding view model.

Reply:
done